rca/static_src/sass/components/_logo.scss

```diff
@@ -3,7 +3,6 @@
     $custom-breakpoint-min: 1023px;
     $custom-breakpoint-max: 1220px;
     display: block;
-    transition: opacity $transition;
```

Comment:
Looks like this wasn't doing anything, as the element class receives the opacity change rather than the parent.

I've moved it so that we get the transition.